packages/esm-patient-forms-app/src/hooks/use-forms.ts

```diff
@@ -1,13 +1,15 @@
 import dayjs from 'dayjs';
 import useSWR from 'swr';
-import { openmrsFetch } from '@openmrs/esm-framework';
+import { openmrsFetch, useConfig } from '@openmrs/esm-framework';
 import { ListResponse, FormEncounter, EncounterWithFormRef, CompletedFormInfo } from '../types';
 import { customEncounterRepresentation, formEncounterUrl } from '../constants';
 import { isFormFullyCached } from '../offline-forms/offline-form-helpers';
 
 export function useFormEncounters(cachedOfflineFormsOnly = false) {
-  return useSWR([formEncounterUrl, cachedOfflineFormsOnly], async () => {
-    const res = await openmrsFetch<ListResponse<FormEncounter>>(formEncounterUrl);
+  const config = useConfig();
```

Comment:
Not sure what you mean by "declare the config." This all looks reasonable though. `useConfig` is a hook, so it's subject to the normal [rules of hooks](https://reactjs.org/docs/hooks-rules.html); since `useFormEncounters` is also a hook, this is usage is good.